app/src/main/java/ch/epfl/sdp/peakar/utils/UIUtils.java

```diff
@@ -0,0 +1,32 @@
+package ch.epfl.sdp.peakar.utils;
+
+public class UIUtils {
+
+    private final static int INTEGER_CHUNK = 3;
+    private final static String SPACE = " ";
+
+    /**
+     * Convert an given integer into a string follow the pattern of
+     * 12 345 678
+     * @return String representing int.
+     */
```

Comment:
Parameter dscription is missing
```suggestion
     * @param number. Integer number to convert to string
     */
```